lib/i18n/index.ts

```diff
@@ -38,15 +38,6 @@ export function configure() {
     .init({
       fallbackLng: 'en',
       supportedLngs: ['en', 'es', 'fr', 'ar', 'tr', 'pt', 'de'],
-      resources: {
```

Comment:
Not sure why but this seemed like it was needed for tests but now it just overwrites the resources loaded from the resource loader plugin.